### PR TITLE
 event_staff

### PR DESCRIPTION
**Pull request by Albin-John**
Module name: event.staff
Specification
New model:
• event.staff.role
◦ name → char
• event.staff
◦ contact→ m2o (res.partner)
◦ role → m2o (event.staff.role)

### CODE REVIEW

File: event_staff/models/event.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        comodel_name="event.staff", inverse_name="event_id", string="Staffs",
+    )
+    staff_count = fields.Integer(
+        compute="_compute_staff_count", string="Total event staffs", store=True,
+    )
+
+    @api.depends("staff_ids")
+    def _compute_staff_count(self):
+        for event in self:
+            event.staff_count = len(event.staff_ids)
```
Comment (eicher31) [inline event_staff/models/event.py:18]: could you add empty EOF

File: event_staff/views/event_staff_role_view.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        <field name="model">event.staff.role</field>
+        <field name="arch" type="xml">
+            <form string="Event Staff Role">
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="name" />
+                        </group>
+                        <group>
+                        </group>
```
Comment (eicher31) [inline event_staff/views/event_staff_role_view.xml:15]: Remove group?

File: event_staff/static/description/index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<section class="oe_container">
```
Comment (eicher31) [inline event_staff/static/description/index.html:1]: Can we remove this file?

**Comment by eicher31** [comment]
Thanks for your work. I just send you a mail with some request.

File: event_staff/views/event_staff_role_view.xml (hunk 2 above)
Comment (Albin-John) [reply event_staff/views/event_staff_role_view.xml:15]: For correct alignment it is necessary.

File: event_staff/static/description/index.html (hunk 3 above)
Comment (Albin-John) [reply event_staff/static/description/index.html]: Yes, Removed [d0a3c2a](https://github.com/Albin-John/event/commit/d0a3c2a58690535313aacd09cdf74a134f837073)

**Review by eicher31 — APPROVED** [review]

**Pull request merged**